src/components/common/markets/clob-widget/context/index.tsx

```
@@ -0,0 +1,284 @@
+    sharesAmount,
+    sharesOwned,
+    strategy,
+  ])
+
+  const { yesPrice, noPrice } = useMemo(() => {
+    if (orderBook) {
+      if (strategy === 'Buy') {
+        const yesPrice = orderBook?.asks.sort((a, b) => a.price - b.price)[0]?.price * 100
+        const noPrice = (1 - orderBook?.bids.sort((a, b) => b.price - a.price)[0]?.price) * 100
+        return {
+          yesPrice: isNaN(yesPrice) ? 0 : +yesPrice.toFixed(),
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)